assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSelectionPanel/index.test.js

```diff
@@ -563,6 +565,188 @@ describe( 'AudienceSelectionPanel', () => {
 		} );
 	} );
 
+	describe( 'AudienceCreationNotice / AudienceCreationSuccess', () => {
+		it( 'should display an audience creation notice with both audiences', async () => {
+			const nonSiteKitConfiguredAudiences = [
+				'properties/12345/audiences/1',
+				'properties/12345/audiences/2',
+			];
+
+			const nonSiteKitReportOptions = {
+				...reportOptions,
+				dimensionFilters: {
+					audienceResourceName: availableAudiences
+						.filter(
+							( { audienceType } ) =>
+								audienceType !== 'SITE_KIT_AUDIENCE'
+						)
+						.map( ( { name } ) => name ),
+				},
+			};
+
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.setAvailableAudiences(
+					availableAudiences.filter(
+						( { audienceType } ) =>
+							audienceType !== 'SITE_KIT_AUDIENCE'
+					)
+				);
```

Comment:
This can be simplified a bit, again I'll apply this myself:

```suggestion
			const nonSiteKitAvailableAudiences = availableAudiences.filter(
				( { audienceType } ) => audienceType !== 'SITE_KIT_AUDIENCE'
			);

			const nonSiteKitConfiguredAudiences =
				nonSiteKitAvailableAudiences.map( ( { name } ) => name );

			const nonSiteKitReportOptions = {
				...reportOptions,
				dimensionFilters: {
					audienceResourceName: nonSiteKitConfiguredAudiences,
				},
			};

			registry
				.dispatch( MODULES_ANALYTICS_4 )
				.setAvailableAudiences( nonSiteKitAvailableAudiences );
```